Cargo.toml

```diff
@@ -18,6 +18,7 @@ name = "cairo"
 [features]
 embed-lgpl-docs = ["gtk-rs-lgpl-docs"]
 png = ["cairo-sys-rs/png"]
+xlib = ["cairo-sys-rs/xlib"]
```

Comment:
I think we should drop this feature from here because there's no way for the crate to provide any Xlib support and you have to explicitly depend on the sys crate to use the functions added by this PR.